### ISSUE TITLE
I owe you a pizza

### ISSUE THREAD

**Issue by TheJaredWilcurt** [devedse/WebOptimizationProject#2]
So everytime I find someone that can losslessly compress a PNG better than me I buy them a pizza in exchange for their secrets.

Here's my current setup, you may be able to squeeze some more bytes out of an image on top of your current process.

* https://www.reddit.com/r/webdev/wiki/optimization#wiki_png_compression_instructions

Gimme an email address and I'll send you a papa johns gift card. :pizza:

**Comment by devedse** [comment]
Haha :smile:, that's amazing!!

Basically I'm using a tool created by nikkho from [Encode.RU - FileOptimizer](https://encode.ru/threads/1589-FileOptimizer/) which in it's turn uses a number of the tools you talk about in your post but also some additional ones.

My tool then does a pixel by pixel comparison to ensure that the image data is still 100% the same, if it is, it will be included in a PullRequest.

So basically, I'm not sure if I should be the one receiving your pizza or nikkho who wrote FileOptimizer :smile:.

Also checkout the readme @ https://github.com/devedse/WebOptimizationProject as it described all information you need to also compress images like that.

**Comment by TPS** [comment]
@javiergutierrezchamorro ([nikkho](https://sourceforge.net/u/nikkho/) @ [sourceforge](https://sourceforge.net/projects/nikkhokkho/) 👋), you should claim your pizza.  Also, check out this amazing derivative of your project.

**Comment by javiergutierrezchamorro** [comment]
Thanks @devedse @TPS 
Since FileOptimizer is also using third party tools (plugins), I guess the pizza should go to their authors.
Anyway donations and code contributions are welcome. :-)

**Comment by devedse** [comment]
@TPS , I'm currently working on something else that might interest you, can you add me on discord so I can show you some things? Devedse#5459

**Comment by TPS** [comment]
> @TPS  can you add me on discord? Devedse#5459

Sadly, I don't happen to Discord.

**Comment by devedse** [comment]
@TPS ah thats too bad. Any other way to PM you?

**Comment by TPS** [comment]
[Try me @ the F-Droid forum.](https://forum.f-droid.org/u/tps)

**Comment by devedse** [comment]
@TPS , created an account there but can't find any way to send PM's.

Maybe you could join Gitter: https://gitter.im/WebOptimizationProject/Lobby?source=orgpage

You can login using your github credentials.

**Comment by TPS** [comment]
@devedse I just PMed you @ F-Droid forum.
